sections/futures/MobileTrade/UserTabs/TransfersTab.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+import { useRecoilValue } from 'recoil';
+import { currentMarketState } from 'store/futures';
+
+import useGetFuturesMarginTransfers from 'queries/futures/useGetFuturesMarginTransfers';
+import { GridDivCenteredRow } from 'styles/common';
+import Table from 'components/Table';
+import { timePresentation } from 'utils/formatters/date';
+
+const TransfersTab: React.FC = () => {
+	const marketAsset = useRecoilValue(currentMarketState);
+
+	const marginTransfersQuery = useGetFuturesMarginTransfers(marketAsset);
+	const marginTransfers = React.useMemo(
+		() => (marginTransfersQuery.isSuccess ? marginTransfersQuery?.data ?? [] : []),
+		[marginTransfersQuery.isSuccess, marginTransfersQuery.data]
+	);
+
+	const { isLoading, isFetched: isLoaded } = marginTransfersQuery;
+
+	const { t } = useTranslation();
+	const columnsDeps = React.useMemo(() => [marginTransfers], [marginTransfers]);
+
+	return (
+		<StyledTable
+			highlightRowsOnHover
+			columns={[
+				{
+					Header: (
+						<StyledTableHeader>{t('futures.market.user.transfers.table.action')}</StyledTableHeader>
+					),
+					accessor: 'action',
+					Cell: (cellProps: any) => <StyledActionCell>{cellProps.value}</StyledActionCell>,
+					width: 50,
+				},
+				{
+					Header: (
+						<StyledTableHeader>{t('futures.market.user.transfers.table.amount')}</StyledTableHeader>
+					),
+					accessor: 'amount',
+					sortType: 'basic',
+					Cell: (cellProps: any) => (
+						<StyledAmountCell isPositive={cellProps.row.original.isPositive}>
+							{cellProps.value}
+						</StyledAmountCell>
+					),
+					sortable: true,
+					width: 50,
+				},
+				{
+					Header: (
+						<StyledTableHeader>{t('futures.market.user.transfers.table.date')}</StyledTableHeader>
+					),
+					accessor: 'timestamp',
+					Cell: (cellProps: any) => (
+						<DefaultCell>{timePresentation(cellProps.value, t)}</DefaultCell>
```

Comment:
This is only successful for my first few transfers:
<img width="376" alt="image" src="https://user-images.githubusercontent.com/10401554/172486877-6eee38c0-0ae9-4bbd-8495-311e2ceafe49.png">

Reply:
Everything on following pages shows a date